bowling/src/main.cpp

```diff
@@ -18,8 +25,47 @@ int main(int argc, char* argv[]) {
     auto inputDirectory = ap.getInputDirectory();
     auto outputFileName = ap.getOutputFileName();
 
-    std::cout << "input dir: " << inputDirectory << "\n";
-    std::cout << "output file: " << outputFileName << "\n";
+    ConsoleStream consoleStr;
+    FileStream* fileStr{nullptr};
```

Comment:
Connected with shared pointer comment above. We can move "ownership" to shared ptr without a lost :)